classes/class-wpcom-liveblog-entry.php

```diff
@@ -23,7 +23,7 @@ public function __construct( $comment ) {
 		if ( $this->replaces && $this->get_content() ) {
 			$this->type = 'update';
 		}
-		if ( 'liveblog_trashed' === $comment->comment_approved ) {
+		if ( '' === $this->get_content() ) {
```

Comment:
This might not be needed if the functionality for deleting entries has been refactored, I'll have a look.